Editor/EditorCore/PolyShapeEditor.cs

```diff
@@ -444,6 +457,10 @@ void DoPointPlacement()
 
                         Handles.color = Color.white;
                     }
+
+                    if(evt.type != EventType.Repaint)
```

Comment:
If you don't want to trigger a SceneView.RepaintAll each frame, I would include EventType.Layout in there. IMGUI "frame" are always composed of Layout+Repaint+other events. Otherwise this could be switched to only repaint all on layout.